### ISSUE TITLE
Coverage for `master` stuck at 85% in codecov

### ISSUE THREAD

**Issue by PawelLipski** [PawelLipski/git-machete#1042]
https://app.codecov.io/gh/VirtusLab/git-machete/tree/master/git_machete

Once fixed, update the badge URL in README.md to point to `master`, not `develop`

**Comment by PawelLipski** [comment]
Huh judging by [logs from master build in CI](https://app.circleci.com/pipelines/github/VirtusLab/git-machete/4284/workflows/e733a68b-4d80-475f-961c-1343ca2a5da6/jobs/24289), the upload passes succesfully:

```
[2023-08-27T18:23:21.977Z] ['info'] Detected CircleCI as the CI provider.
[2023-08-27T18:23:21.978Z] ['info'] Pinging Codecov: https://codecov.io/upload/v4?package=codecov-circleci-orb-3.2.2-uploader-0.6.2&token=*******&branch=master&build=24289&build_url=https%3A%2F%2Fcircleci.com%2Fgh%2FVirtusLab%2Fgit-machete%2F24289&commit=703f5c6f913fcfc76d61a8f0a847dadc5badbb8c&job=0&pr=&service=circleci&slug=VirtusLab%2Fgit-machete&name=24289&tag=&flags=&parent=
[2023-08-27T18:23:22.167Z] ['info'] https://app.codecov.io/github/VirtusLab/git-machete/commit/703f5c6f913fcfc76d61a8f0a847dadc5badbb8c
https://storage.googleapis.com/codecov/v4/raw/2023-08-27/826F8A8F9120D45AE9A4BE5DF81599E7/703f5c6f913fcfc76d61a8f0a847dadc5badbb8c/a2047c3e-e355-4a18-bf39-2001100f37c3.txt?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=GOOG1EJOGFN2JQ4OCTGA2MU5AEIT7OT5Z7HTFOAN2SPG4NWSN2UJYOY5U6LZQ%2F20230827%2FUS%2Fs3%2Faws4_request&X-Amz-Date=20230827T182322Z&X-Amz-Expires=10&X-Amz-SignedHeaders=host&X-Amz-Signature=585df5d1daf8ba2cdfb9bd784db5471a9018eff1519b1a100fa14f78f943e688
[2023-08-27T18:23:22.167Z] ['info'] Uploading...
[2023-08-27T18:23:22.319Z] ['info'] {"status":"success","resultURL":"https://app.codecov.io/github/VirtusLab/git-machete/commit/703f5c6f913fcfc76d61a8f0a847dadc5badbb8c"}
```

The problem is that the commit in question (`703f5c6f913fcfc76d61a8f0a847dadc5badbb8c`) has already been built & coverage uploaded on `develop` as well... and apparently Codecov [associates this commit with `develop` branch](https://app.codecov.io/github/VirtusLab/git-machete/commit/703f5c6f913fcfc76d61a8f0a847dadc5badbb8c):

<img width="742" alt="image" src="https://github.com/VirtusLab/git-machete/assets/3383210/f6904825-cc30-41c1-bf09-eaf24a4d5840">

We could skip uploading coverage on `develop` builds for `master` builds to be recorded correctly... but I'd say that coverage on `develop` is more important.

So let's leave the issue as is, and wait until Codecov fixes the bug. Actually, it seems to be a regression... after all, it used to be computed correctly back until around our version 3.7.2 (the last recorded master commit is [Apply minor fixes before releasing v3.7.2](https://app.codecov.io/gh/VirtusLab/git-machete/commit/862c182f83c502c89a8f6f7e9a9107e18626c793)).

**Comment by PawelLipski** [comment]
Looks fixed somehow 😅 quite a long time ago, apparently:

<img width="1308" alt="image" src="https://github.com/VirtusLab/git-machete/assets/3383210/89f4716a-239f-4949-b740-3e910afd4db6">

coz there's no change in the last 3 months.
